labs/showcase.md

```diff
@@ -183,7 +183,7 @@ Below are some of the projects that our collaborators are working on right now.
 
 Interested in proposing a new arXiv Labs project?
 
- <a onclick="pendo.feedback.openFeedback(event)" class="button-fancy">Click to view existing proposals & submit new ideas <span> </span></a>
+<a onclick="/project-proposal" class="button-fancy">Click to submit your idea <span> </span></a>
```

Comment:
Is span needed?

Reply:
I know it looks empty in the html. But the span is used to display and animate the right double chevron you see on the live site.